bundles/org.openhab.ui/web/src/css/app.styl

```diff
@@ -140,6 +140,15 @@ html
    .delete
      --f7-bars-link-color #ff3b30
 
+// Fix actions-modal/popup is placed behind the notch or outside of the screen on iPhone in openHAB iOS app
+@media (orientation: portrait)
+  .ios .actions-modal
+    max-height: calc(100% - env(safe-area-inset-top, 0px) - env(safe-area-inset-bottom, 0px))
+
+@media (orientation: landscape)
+  .ios .actions-modal
+    max-height: calc(100% - env(safe-area-inset-left, 0px))
```

Comment:
Are you sure there's a need for two different implementations for portrait & landscape?
And it doesn't seem right to use the left safe area to configure a height (when you rotate into landscape mode the top/bottom safe areas should automatically reflect the new orientation, see https://webkit.org/blog/7929/designing-websites-for-iphone-x/)

Reply:
Thanks for the hint. I've must overread that.